blog/new-co-chair/index.qmd

```diff
@@ -0,0 +1,19 @@
+---
+title: "New European Working Group Co-Chair"
+subtitle: "Announcing a new working group co-chair in the European region"
+author: "Alessandro"
+date: "2024-04-19"
+categories: [news]
+image: "openstatsware-hex-300.png"
+---
+
+Hi everyone,
+
+Alessandro here.
+I am writing to announce that later this year I am taking up the role of co-chair for the `openstatsware` working group.
```

Comment:
```suggestion
I am writing to announce that I am taking up the role of co-chair for the `openstatsware` working group.
```
Because it is from now on already:)